test/e2e/storage/testsuites/provisioning.go

```diff
@@ -433,9 +435,12 @@ func PVMultiNodeCheck(client clientset.Interface, claim *v1.PersistentVolumeClai
 	framework.ExpectNoError(framework.WaitForPodSuccessInNamespaceSlow(client, pod.Name, pod.Namespace))
 	runningPod, err = client.CoreV1().Pods(pod.Namespace).Get(pod.Name, metav1.GetOptions{})
 	framework.ExpectNoError(err, "get pod")
-	gomega.Expect(runningPod.Spec.NodeName).NotTo(gomega.Equal(actualNodeName), "second pod should have run on a different node")
 	StopPod(client, pod)
 	pod = nil
+	// The second pod got scheduled on the same node as the first one: skip the test.
```

Comment:
I think you should still drop a warning that it landed on the same node.  

/assign @timothysc

Reply:
OK. Will do.